engines/sci/engine/script_patches.cpp

```diff
@@ -3684,12 +3684,42 @@ static const uint16 phant1RatPatch[] = {
 	PATCH_END
 };
 
+// Phantasmagoria will have the pointer color be black when hovering it over 
+// a button for the first time, when quiting from the main menu and selecting no 
+// or quiting while in game. The pointer color should be red over buttons.
+// The HandleEvent was being trigger, but a check of global193==0 was false 
+// in HotSpot::doit() (since it is 2) so it leaves the function where the 
+// cursor color gets set (SetCel). This condition check has been removed.
+// If the signature is shortened too much it may replace some other part
+// of the code that matches.
+// Applies to at least: English CD
+static const uint16 phant1RedQuitPointerSignature[] = {
+	0x18,                         // not
+	SIG_MAGICDWORD,
+	0x31, 0x25,                   // bnt 25 [00e9]
+	0x89, 0xc1,                   // lsg c1
+	0x35, 0x02,                   // ldi 02
+	0x1c,                         // ne?
+	0x31, 0x12,                   // bnt 12 [00dd]
+	0x39, 0x0f,                   // pushi 0f
+	0x78,                         // push1
+	0x76,                         // push0
+	0x38, 0x4c, 0x01,             // pushi 014c
+	SIG_END
+};
+
+static const uint16 phant1RedQuitPointerPatch[] = {
+	0x33, 0x08,                   // jmp [past global193 check]
```

Comment:
This patch is overwriting the end of the `((not (gDButton respondsTo: #exitDir))` check, because the signature starts at `not, bnt`, not at `lsg c1`. Either use the PATCH_ADDTOOFFSET macro to forward to the correct place to apply the patch (relative to the start of the signature), or remove the unnecessary extra bytes in the signature prior to the actual place we want to rewrite.